apps/crn-server/test/handlers/webhooks/webhook-run-migrations.test.ts

```diff
@@ -182,15 +182,15 @@ describe('Run-migrations Webhook', () => {
       const squidexCreateSpy = jest.spyOn(SquidexRest.prototype, 'create');
 
       await expect(runHandler(...mockHandlerArguments)).rejects.toThrow();
-      expect(mockUp).toHaveBeenCalledTimes(2);
+      expect(mockUp).toHaveBeenCalledTimes(1);
```

Comment:
ok so the goal of the test is not really preserved actually. You see - the point of this test is this:

1. Our read-dir function finds 3 files and they need to be run in sequence
2. The runner should run the first one - it should not error as per this line:
https://github.com/yldio/asap-hub/pull/2545/files#diff-430f88e7bf62e1cd44dc77f7cce896503065c4af36cc1000bec709525bbdc7b9R179
3. Once processed the runner should store the information in the db that the file has been run (see the expectation on the spy below)
4. The runner should then proceed to the next migration by running the mockUp function again and this should fail as per 
https://github.com/yldio/asap-hub/pull/2545/files#diff-430f88e7bf62e1cd44dc77f7cce896503065c4af36cc1000bec709525bbdc7b9R180
5. The runner should exit by throwing but before it should log that the second migration has failed (see expectation at line #191)

Now what I think happens is that the new jest somehow does not preserve the order of rejected and resolved values on the mockUp - it's worth checking if there is a way to tell it to do it in such order.